src/NuGet.Core/NuGet.Protocol/Model/LocalPackageSearchMetadata.cs

```diff
@@ -85,13 +89,17 @@ private static Uri Convert(string uri)
 
         public bool IsListed => true;
 
-        // The prefix reservation is not applicable to local packages
+        /// <remarks>
+        /// The prefix reservation is not applicable to local packages
+        /// </remarks>
         public bool PrefixReserved => false;
 
         public LicenseMetadata LicenseMetadata => _nuspec.GetLicenseMetadata();
 
-        // Deprecation metadata is not stored within the package and requires an online package source.
-        public PackageDeprecationMetadata DeprecationMetadata => null;
+        /// <remarks>
+        /// Deprecation metadata is not stored within the package and requires an online package source.
+        /// </remarks>
+        public Task<PackageDeprecationMetadata> GetDeprecationMetadataAsync() => Task.FromResult<PackageDeprecationMetadata>(null);
```

Comment:
You can write a test for this.

Reply:
There were no existing tests for this class, so I made a simple one to check just this property.